packages/grafana-ui/src/themes/default.ts

```diff
@@ -123,15 +135,7 @@ const theme: GrafanaThemeCommons = {
   },
   panelPadding: 8,
   panelHeaderHeight: 28,
-  zIndex: {
-    navbarFixed: 1000,
-    sidemenu: 1020,
-    dropdown: 1030,
-    typeahead: 1030,
-    tooltip: 1040,
-    modalBackdrop: 1050,
-    modal: 1060,
-  },
+  zIndex: appZIndexes,
```

Comment:
this is the old theme, new theme we introduced in v8 is here https://github.com/grafana/grafana/blob/main/packages/grafana-data/src/themes/zIndex.ts  ,  think we missed making this use properties from the new theme